tools/KeyfileIOUtils.cpp

```diff
@@ -63,6 +73,21 @@ static void validateRSAPrivateKey(const std::string& key) {
 
   if (!std::regex_match(key, std::regex("[0-9A-Fa-f]+"))) throw std::runtime_error("Invalid RSA private key: " + key);
 }
+#elif USE_EDDSA_OPENSSL
+static void validateEdDSAPublicKey(const std::string& key) {
+  const size_t eddsaPublicKeyHexadecimalLength{64UL};
```

Comment:
please use a named constant in all of the validation functions

Reply:
Changed to `EdDSAPublicKeyByteSize * 2`.